apps/test/unit/templates/studioHomepages/TeacherHomepageTest.js

```diff
@@ -66,14 +58,15 @@ describe('TeacherHomepage', () => {
         isRtl={false}
       />
     );
-    const headerBanner = wrapper.childAt(0);
-    assert.equal(headerBanner.name(),'HeaderBanner');
-    assert.equal(headerBanner.props().headingText, 'Home');
-    assert.equal(headerBanner.props().extended, false);
+    const headerBanner = wrapper.find('HeaderBanner');
+    assert.deepEqual(headerBanner.props(), {
+      headingText: "Home",
+      short: true
+    });
```

Comment:
I've been wondering if it's nicer to express these as 
```jsx
expect(wrapper).to.containMatchingElement(
  <HeaderBanner
    headingText="Home"
    short
  />
);
```

I think it's a good assertion either way, I'll leave it up to you.